README.md

```diff
@@ -161,6 +161,39 @@ FreshRSS supports further customizations by adding extensions on top of its core
 See the [repository dedicated to those extensions](https://github.com/FreshRSS/Extensions). 
 
 
+# APIs & native apps
+
+FreshRSS supports access via native apps for Linux, Android, iOS and OS X.
+
+## Google Reader-like API
+
+There is more information available about our Google Reader compatible API on the page [mobile access](https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html).
+
+Supported clients are:
+
+* Android
+	* [News+](https://play.google.com/store/apps/details?id=com.noinnion.android.newsplus) with [News+ Google Reader extension](https://play.google.com/store/apps/details?id=com.noinnion.android.newsplus.extension.google_reader) (Closed source)
+	* [FeedMe 3.5.3+](https://play.google.com/store/apps/details?id=com.seazon.feedme) (Closed source)
+	* [EasyRSS](https://github.com/Alkarex/EasyRSS) (Open source, [F-Droid](https://f-droid.org/packages/org.freshrss.easyrss/))
+* GNU/Linux
+	* [FeedReader 2.0+](https://jangernert.github.io/FeedReader/) (Open source)
+
+## Fever API
+
+You can find more information about this API at our [Fever API documentation](https://freshrss.github.io/FreshRSS/en/users/06_Fever_API.html) page.
+
+Supported clients are:
+
+- iOS
+  - Fiery Feeds
+  - Unread
+
+- MacOS
+  - Readkit
```

Comment:
Seems to be https://readkitapp.com.
Same as above, the service seems to already support Google Reader API.

Reply:
How can you see that? The APP has nothing in "add account" that sounds like Google Reader. Does one of the supported services use GReade API in the background?

Reply:
It supports e.g. Feedly, which is using the Google Reader API. I do not have anything Apple around, so could you please check for which services Fiery Feeds gives the user the option to type a server address?

Reply:
Instapaper: Username/Password
Pocket: OAuth
Pinboard: Username/Password
Feedly: OAuth
Fever: Domain/Email/Password
NewsBlur: Username/Password
Feedwrangler: Email/Password
Feedbin: Email/Password